contrib/submit-protocol/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "submit-protocol",
+  "version": "1.0.0",
+  "description": "A web portal plugin to submit protocol.",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "webpack-dev-server --mode=development",
+    "prebuild": "tslint --project .",
+    "build": "webpack --mode=production"
+  },
+  "author": "Microsoft Corporation",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/pai.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Microsoft/pai/issues"
+  },
+  "homepage": "https://github.com/Microsoft/pai#readme",
+  "license": "MIT",
+  "dependencies": {
+    "bootstrap": "3.4.0",
+    "classnames": "^2.2.6",
+    "immutability-helper": "^3.0.0",
+    "js-yaml": "^3.13.0",
+    "monaco-editor": "^0.16.2",
+    "office-ui-fabric-react": "^6.162.1",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "react-monaco-editor": "^0.25.1"
+  },
+  "devDependencies": {
+    "@types/classnames": "^2.2.7",
```

Comment:
I've found typing files in classnames package itself: https://github.com/JedWatson/classnames/blob/master/index.d.ts

Reply:
the latest [classnames version](https://www.npmjs.com/package/classnames) (10 months ago) does not contain typing files.